pages/_app.tsx

```diff
@@ -96,7 +97,9 @@ const App = ({ Component, pageProps }: AppProps) => {
                         <GraphProvider>
                             <FactoryStore>
                                 <TransactionStore>
-                                    <Component {...pageProps} />
+                                    <OMEStore>
```

Comment:
The insurance page wont need the OMEStore. I recommend just using 
```
    const { filledOrders, refetchFilledOrders } = useUsersMatched(selectedTracer?.address ?? '', account ?? '');
```
as a standalone hook